Configurations/windows-makefile.tmpl

```diff
@@ -773,13 +773,24 @@ EOF
               # If the target is named foo.S in build.info, we want to
               # end up generating foo.s in two steps.
               if ($args{src} =~ /\.S$/) {
-                   return <<"EOF";
+                if(($config{CC} !~ /clang-cl/) and ($config{AS} =~ /clang-cl/))
+                {
+                    return <<"EOF";
```

Comment:
This is unnecessary, as the added `/D__ASSEMBLER__` can be used with cl.exe too, without harm.